<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgHopfenGabe</class>
 <widget class="QWidget" name="WdgHopfenGabe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>125</height>
   </rect>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>4</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QFrame" name="frameColor">
          <property name="minimumSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="focusPolicy">
           <enum>Qt::ClickFocus</enum>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ComboBox" name="cbZutat">
          <property name="focusPolicy">
           <enum>Qt::StrongFocus</enum>
          </property>
          <property name="toolTip">
           <string>Hopfenauswahl.</string>
          </property>
          <property name="maxVisibleItems">
           <number>20</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnLoeschen">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>24</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>22</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Hopfengabe l√∂schen.</string>
          </property>
          <property name="icon">
           <iconset resource="../images.qrc">
            <normaloff>:/images/close_button.svg</normaloff>:/images/close_button.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <property name="verticalSpacing">
         <number>4</number>
        </property>
        <item row="0" column="4" rowspan="2">
         <widget class="QLabel" name="lblMengeProzent">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>%</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5" rowspan="2">
         <widget class="QPushButton" name="btnMengeKorrektur">
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Korrektur, um auf 100% zu kommen.</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="SpinBox" name="tbMenge">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Hopfenmenge.</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="maximum">
           <number>999</number>
          </property>
         </widget>
        </item>
        <item row="0" column="2" rowspan="2" colspan="2">
         <widget class="QLabel" name="label_6">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>g</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Anteil</string>
          </property>
         </widget>
        </item>
        <item row="2" column="7">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Alpha</string>
          </property>
         </widget>
        </item>
        <item row="1" column="8">
         <widget class="SpinBox" name="tbVorhanden">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Vorhandene Menge.</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="maximum">
           <number>9999</number>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="DoubleSpinBox" name="tbMengeProzent">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Prozentanteil am Gewicht.</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="minimum">
           <double>0.000000000000000</double>
          </property>
          <property name="maximum">
           <double>100.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="DoubleSpinBox" name="tbAnteil">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Anteil an Bittere.</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="maximum">
           <double>999.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="7" rowspan="2">
         <widget class="QLabel" name="lblVorhanden">
          <property name="text">
           <string>Vorhanden</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QLabel" name="lblAnteilProzent">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>%</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="label_13">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>IBU</string>
          </property>
         </widget>
        </item>
        <item row="2" column="5">
         <widget class="QPushButton" name="btnAnteilKorrektur">
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Korrektur, um auf 100% zu kommen.</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="DoubleSpinBox" name="tbAnteilProzent">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Prozentanteil an Bittere.</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="minimum">
           <double>0.000000000000000</double>
          </property>
          <property name="maximum">
           <double>100.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="9" rowspan="2">
         <widget class="QLabel" name="lblEinheit">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>g</string>
          </property>
         </widget>
        </item>
        <item row="2" column="8">
         <widget class="DoubleSpinBox" name="tbAlpha">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Alphas√§ure des Hopfens.</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="maximum">
           <double>999.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="6" rowspan="2">
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="9">
         <widget class="QLabel" name="label_9">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>%</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" rowspan="2">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Menge</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Kochdauer</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="SpinBox" name="tbKochdauer">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Kochdauer.</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="minimum">
           <number>-999</number>
          </property>
          <property name="maximum">
           <number>999</number>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLabel" name="label_12">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>min</string>
          </property>
         </widget>
        </item>
        <item row="3" column="9">
         <widget class="QLabel" name="label_10">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>%</string>
          </property>
         </widget>
        </item>
        <item row="3" column="8">
         <widget class="DoubleSpinBox" name="tbAusbeute">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Alphas√§ureausbeute.</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="maximum">
           <double>999.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
        <item row="3" column="7">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Ausbeute</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3" colspan="3">
         <widget class="QComboBox" name="cbZeitpunkt">
          <property name="toolTip">
           <string>Zeitpunkt der Hopfengabe.</string>
          </property>
          <item>
           <property name="text">
            <string>Vorderw√ºrze</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Kochen</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Nachisomerisierung</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>beim Ausschlagen</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>widgets/combobox.h</header>
  </customwidget>
  <customwidget>
   <class>DoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>widgets/doublespinbox.h</header>
  </customwidget>
  <customwidget>
   <class>SpinBox</class>
   <extends>QSpinBox</extends>
   <header>widgets/spinbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cbZutat</tabstop>
  <tabstop>btnLoeschen</tabstop>
  <tabstop>tbMenge</tabstop>
  <tabstop>tbMengeProzent</tabstop>
  <tabstop>btnMengeKorrektur</tabstop>
  <tabstop>tbVorhanden</tabstop>
  <tabstop>tbAnteil</tabstop>
  <tabstop>tbAnteilProzent</tabstop>
  <tabstop>btnAnteilKorrektur</tabstop>
  <tabstop>tbAlpha</tabstop>
  <tabstop>tbKochdauer</tabstop>
  <tabstop>cbZeitpunkt</tabstop>
  <tabstop>tbAusbeute</tabstop>
 </tabstops>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
